Microsoft.NET.Build.Containers/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -76,6 +76,7 @@
                         WorkingDirectory="$(ContainerWorkingDirectory)"
                         Entrypoint="@(ContainerEntrypoint)"
                         EntrypointArgs="@(ContainerEntrypointArgs)"
-						Labels="@(ContainerLabel)"/>
+                        Labels="@(ContainerLabel)"
+                        ExposedPorts="@(ContainerPort)" />
```

Comment:
Nit: We may want to rename `ContainerPort` to `ContainerPorts`.

Reply:
excellent point 👍

Reply:
Thinking about this more I don't think I agree - each individual MSBuild Item is a single `ContainerPort`.